xbmc/Application.h

```diff
@@ -255,8 +255,9 @@ class CApplication : public CXBApplicationEx, public IPlayerCallback, public IMs
   /*!
    \brief Starts a video library cleanup.
    \param userInitiated Whether the action was initiated by the user (either via GUI or any other method) or not.  It is meant to hide or show dialogs.
+   \param content Content type to clean, blank for everything
    */
-  void StartVideoCleanup(bool userInitiated = true);
+  void StartVideoCleanup(bool userInitiated = true, const std::string& content="");
```

Comment:
Would you mind putting a blank around `=`? Don't know whether this is part of our coding guidelines, but this is how we usually format our code.

Reply:
not at all. added.

Reply:
man, you are pretty fast!